lib/tasks/verify.js

```diff
@@ -177,5 +177,5 @@ module.exports.origamiJson = function() {
 	return result;
 };
 
-module.exports.watchable = true;
+module.exports.watchable = false;
```

Comment:
It's not really useful as the watching stops as soon as there's a linting
error. I implemented the cleverer watch there, and then figured out that
verify --watch is just not useful

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.